### PR TITLE
Lots of misc gui fixes

### LINKED ISSUE #440: Factory Reset button in Serial Console could have a pop up warning about what it does
Users tend to click everything, and this could be one where we want to caution them not to just push forward blindly. Calibration and wi-fi creds get deleted with this button, so it could be a problem.

### PR DESCRIPTION
**Pull request by ImUrX**
- Fixes #440, #510
- Remove unneeded dependencies like ``fs-nextra`` and ``dotenv``
- Fixed flag emojis not working in Linux (**requires testing on Windows!**)
- ``Typography`` supports whitespace now specification which is nice for newline detection in translations

### CODE REVIEW

**Comment by ImUrX** [comment]
Example images:
neck
serial

**Review by Eirenliel — APPROVED** [review]

**Review by Louka3000 — CHANGES_REQUESTED** [review]
You're only making the neck warning appear in the "manage all trackers" tracker assignment menu. Please make it appear in the individual tracker assignment popup as well (when you click on a tracker and assign its body position from there)

**Review by Louka3000 — APPROVED** [review]
I added furry and french translation

**Comment by ImUrX** [comment]
last commit made position be able to be converted to cm precision

File: gui/src/components/onboarding/NeckWarningModal.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  setShowed: (arg0: boolean) => void;
+} & ReactModal.Props) {
+  const { l10n } = useLocalization();
+  // Skip popup if bodyPart isn't neck
+  if (isOpen && !hasShowed && bodyPart !== BodyPart.NECK) {
+    setShowed(true);
+  }
+  // Reset when no longer opened
+  if (!isOpen && hasShowed) {
+    setShowed(false);
```
Comment (ButterscotchV) [inline gui/src/components/onboarding/NeckWarningModal.tsx:29]: Misspelled "Showed" should be "Shown" (Not blocking merge for this).

File: gui/src/components/onboarding/NeckWarningModal.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      className={props.className}
+      overlayClassName={props.overlayClassName}
+    >
+      <div className="flex w-full h-full flex-col ">
+        <div className="flex w-full flex-col flex-grow items-center gap-3">
+          <Localized
+            id="tracker_selection_menu-neck_warning"
+            elems={{ b: <b></b> }}
+          >
+            <WarningBox>Warning!</WarningBox>
```
Comment (ButterscotchV) [inline gui/src/components/onboarding/NeckWarningModal.tsx:47]: `Warning!` should be translated.

File: gui/src/components/settings/pages/Serial.tsx [hunk 3]
```diff
@@ -152,67 +158,91 @@ export function Serial() {
...
+          elems={{ b: <b></b> }}
         >
-          <div className="flex select-text px-3">
-            <pre>
-              {isSerialOpen
-                ? consoleContent
-                : l10n.getString('settings-serial-connection_lost')}
-            </pre>
-          </div>
+          <WarningBox>Warning!</WarningBox>
```
Comment (ButterscotchV) [inline gui/src/components/settings/pages/Serial.tsx:170]: Same as last comment about this, `Warning!` should be translated.

**Review by ButterscotchV — CHANGES_REQUESTED** [review]

**Pull request merged**